packages/in17n/package.json

```
@@ -0,0 +1,72 @@
+{
+	"name": "@sapphire/plugin-in17n",
```

```suggestion
	"name": "@sapphire/plugin-i18next",
```

And reflect that change everywhere else as well